CommunityCall/20190516_Notes.md

```diff
@@ -0,0 +1,29 @@
+# PowerShell Community Call - May 16, 2019
+
+* update on PS 7
+    * first preview *should* come out next week
+    * expect a roadmap to coincide with the preview.1 release
+    * all the new .NET Core 3.0 types will be there
+    * Out-GridView will not be part of preview.1
+    * lots of functionality that the didn't make the cut for 6.2
+    * .NET 5 blog post: https://devblogs.microsoft.com/dotnet/introducing-net-5/
+    * PSReadline repo has been officially transitioned to the PowerShell Team
+        * more maintainers from within the PS team to get 2.0 to a GA quality release
+* blogs
+    * [Compatibility rules for PSScriptAnalyzer](https://devblogs.microsoft.com/powershell/using-psscriptanalyzer-to-check-powershell-version-compatibility/)
+    * [PowerShell in Azure Functions - Public Preview](https://devblogs.microsoft.com/powershell/public-preview-of-powershell-in-azure-functions-2-x/)
+* events
+    * Build was a success last week
+    * PSConfEU first week of June
+    * PSConfAsia: September 19-21
+* RFCs
+    * Simpler concurrency parallelism natively supported in PowerShell for people who don't want to manage their own thread jobs: https://github.com/PowerShell/PowerShell-RFC/pull/174
+    * Test-Connection: https://github.com/PowerShell/PowerShell-RFC/pull/172
+    * Support login shell in *nix: https://github.com/PowerShell/PowerShell-RFC/pull/171
```

Comment:
Asterisk needs to be escaped for markdown to be rendered correctly.  `\*` looks weird when read as text, so suggest just using `Unix`.

```suggestion
    * Support login shell in Unix: https://github.com/PowerShell/PowerShell-RFC/pull/171
```